### PR TITLE
Let `isNotVisible()` pass if the element does not exist

### PR DESCRIPTION
**Pull request by scalvert**
Fixing the logic in is-visible/is-not-visible to account for when elements are missing in the DOM.
These assertions would trigger the Element <el> exists assertion that was triggered when an element was not found in the DOM, regardless of correctness
- `is-visible` should **pass** in the case where the element **exists** and is visible
- `is-visible` should **fail** in the case where the element **doesn't exist** or is not visible
- `is-not-visible` should **pass** in the case where the element **doesn't exist** or is not visible
- `is-not-visible` should **fail** in the case where the element **exists** and is visible

### CODE REVIEW

**Comment by scalvert** [comment]
O_o

**Comment by Turbo87** [comment]
looks like this has conflicts now... 😞 

I tend to agree that this logic is probably better. I'll review the code changes hopefully tomorrow.

**Comment by scalvert** [comment]
I’ll fix the conflicts.

**Comment by scalvert** [comment]
@Turbo87 fixed!

File: lib/assertions.js [hunk 1]
```diff
@@ -226,34 +226,54 @@ export default class DOMAssertions {
...
+      let actual =
+        actualValue === null
+          ? `Element ${
+              this.targetDescription
+            } does not have attribute "${name}"`
+          : `Element ${
+              this.targetDescription
+            } has attribute "${name}" with value ${JSON.stringify(
+              actualValue
+            )}`;
```
Comment (Turbo87) [inline lib/assertions.js]: @scalvert this looks like unrelated changes 🤔 

I'm assuming you used prettier here, but I'm not sure if the result in this case is actually prettier than before... 😄

Comment (scalvert) [reply lib/assertions.js]: Damn. It’s vscode. I format on save for some repos and not for others. It’s maddening. Will fix.

**Comment by scalvert** [comment]
@Turbo87 ok I've managed to wrangle this PR into a good state. Can you take another look?

**Comment by Turbo87** [comment]
Merci! :)
